scripts/watch.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+const DEVELOPMENT = 'development';
+process.env.NODE_ENV = DEVELOPMENT;
+
+const fs = require('fs-extra');
+const paths = require('../config/paths');
+const webpack = require('webpack');
+const config = require('../config/webpack.config.js');
+const devConfig = config(DEVELOPMENT);
+
+webpack(devConfig).watch({}, (err, stats) => {
+	if (err) {
+		console.error(err);
+	}
+	console.error(
```

Comment:
Is it supposed to be `console.error()`?

Reply:
Not sure, maybe they wanted to emphasize.
Updated to be `console.log`